tests/run.yml

```diff
@@ -0,0 +1,52 @@
+---
+#Run main playbook
+- hosts: streisand
+  gather_facts: no
+  remote_user: root
+  become: yes
+  pre_tasks:
+    - name:
+      raw: sudo apt update && apt install python -y
+
+- hosts: streisand
+  gather_facts: yes
+  remote_user: root
+  become: yes
+  tasks:
+    - name: Get the default SSH key
```

Comment:
You are 100% right about generating a unique key. We totally could do that. I will update.